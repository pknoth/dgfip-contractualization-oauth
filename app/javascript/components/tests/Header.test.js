import ReactTestRenderer from 'react-test-renderer'
import Header from '../header'

describe('components | Header', () => {
  describe('render', () => {
    const renderer = ReactTestRenderer.create(<Header />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
