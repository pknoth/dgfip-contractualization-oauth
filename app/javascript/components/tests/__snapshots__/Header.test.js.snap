// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | Header render should match snapshot 1`] = `
<header>
  <nav
    className="nav"
  >
    <div
      className="nav__container"
    >
      <a
        className="nav__link"
        href="/"
      >
        <img
          alt="Accueil de particulier.api.data.gouv.fr"
          className="nav__logo"
          src="/static/images/logo-api-particulier.png"
        />
      </a>
      <ul
        className="nav__links"
      >
        <li>
          <a
            href="/about"
            onClick={[Function]}
          >
            A propos
          </a>
        </li>
        <li>
          <a
            href="/documentation"
            onClick={[Function]}
          >
            Documentation technique
          </a>
        </li>
        <li>
          <a
            href="/contact"
            onClick={[Function]}
          >
            Contactez-nous
          </a>
        </li>
      </ul>
      <a
        className="nav__link"
        href="/"
      >
        <img
          alt="Ma sélection de jeux de données"
          className="nav__logo"
          src="../static/images/file-icon.svg"
        />
      </a>
    </div>
  </nav>
</header>
`;
